### ISSUE TITLE
Disable Option [Single Player]

### ISSUE THREAD

**Issue by QORTEC** [lumien231/Hardcore-Darkness#12]
According to your CurseForge page your config file has 3 options:
0. There is no minimum block & sky light
1. There is no minimum block light
2. There is no minimum block light & the sky light is dependent on the moon phase

It also says "If you install this mod on a server all clients will have to have Hardcore Darkness installed. Clients will also be forced to use the config file you specify on your server."

---

And if I understand the above sentence correctly it means when you join a server your client will use the darkness option set on the server [the client settings are ignored].

If that's the case would it be possible to add a option to disable the mod unless you join a server that requires it?

Usecase:
- users/players can chose to disable the mod in single player if they prefer to do so
- server admins can chose the darkness setting to create a certain atmosphere on their server

**Comment by PitchBright** [comment]
If @lumien231 fixes #7 by adding an Option 4, that would solve this too, because you'd be able to set your preferred minimum light level to whatever Mojang sets theirs to.

**Comment by lumien231** [comment]
Not sure **whether** i will will fix that because that person basically just wants the blue hue and nothing else^^. I will probably add a config option that's false by default which only enables the mod on servers that require it.

**Comment by QORTEC** [comment]
I was just re-reading the CurseForge page and noticed that there is a "Dimension Blacklist", is there a wildcard that will select all dimensions?

**Comment by PitchBright** [comment]
@lumien231 are you sure that's what he wanted it for? It seemed to me he wanted the mod disabled, totally, for singleplayer (based on what he wrote here). 

I was going to suggest the obviously which would be "remove the mod", but maybe he doesn't want to go in and out of his modpack folder, or be breaking the modpack up like that.

**Comment by QORTEC** [comment]
@PitchBright correct, "remove the mod" not always the best solution especially when you play both single and multiplayer. [servers with this mod require your client have this mod]

**Comment by PitchBright** [comment]
Remove the mod for singleplayer, is what I was talking about. You said you wanted a way to disable the mod for singleplayer. When I'm playing singleplayer and I don't want a mod to affect me, I do the obvious thing, and remove it from my mods folder.

**Comment by PitchBright** [comment]
Eitherway, I think my original point stands that by adding the feature requested in #7, it would kill two birds with one stone.

**Comment by QORTEC** [comment]
I agree with @lumien231 this request would not fix Issue #7, not the core issue anyway. From what I understand it seams like @FireEgl wants the ability to set a variable for the additional darkness level. This issue/request suggests adding a option to disable the mod entirely; so the mod will be either **off** or **on** [function the way it does no now]. 

If @lumien231 where to decide to solve Issue #7 this this issue may be solved as well [depending on the approach he took]. If you could set the single player to 0% additional darkness then the mod would essentially be disabled and solve this issue.

**Comment by PitchBright** [comment]
Right, although, I certainly wasn't suggesting that solving THIS issue, would fix #7, nor was @lumien231 saying that solving this issue would fix #7. It's the other way around. @lumien231 didn't seem keen to solve #7 because it seemed to him that the only reason that guy requested that particular feature was because he wanted the blue hue.

That said, when I read issue #7… that is totally NOT how I perceived his request, so lumien read it completely differently than I did. I read the issue and took it at face value, so I thought the guy simply wanted to be able to control/config the minLightLevel. It seemed to me that he was just making commentary about the neato-factor of the Blue Hue. It's also the reason I didn't open my own issue for that feature request… it seemed redundant to do so. #7 covered exactly what I was looking for.

Speaking of which, I've never experienced the Blue Hue everybody is talking about… when I've tried all 3 modes. So maybe I'm looking for it in the wrong place. Where is the blue hue?

**Comment by lumien231** [comment]
In any mode that still has sky light turn your gamma up as much as possible and look at something white outside. If you don't have the "alternative" color scheme enabled in the config file it should be blue.

**Comment by PitchBright** [comment]
Okay cool… I see now. Yeah, the blue hue is pretty friggin' cool.
